packages/store/addon/-private/system/promise-proxies.js

```diff
@@ -17,7 +17,7 @@ import { Promise } from 'rsvp';
   updated when the promise resolves.
 
   For more information see the [Ember.PromiseProxyMixin
-  documentation](/api/classes/Ember.PromiseProxyMixin.html).
+  documentation](/api/classes/PromiseProxyMixin.html).
```

Comment:
I'm not sure if this link is correct with `.html` as we actually want it to go to https://api.emberjs.com/ember/release/classes/PromiseProxyMixin

Here is the Discord convo: https://discordapp.com/channels/480462759797063690/480777444203429888/725808587343003720

//

FWIW, I also can't search for PromiseProxyMixin, not sure if that's a legit separate issue?

![image](https://user-images.githubusercontent.com/1372946/85800896-a1252e80-b6f6-11ea-8e1c-84e079a78526.png)